TestON/drivers/common/cli/emulator/mininetclidriver.py

```diff
@@ -1092,49 +1107,130 @@ def getSwController( self, sw ):
             main.cleanup()
             main.exit()
 
-    def assignSwController( self, **kwargs ):
+    def assignSwController( self, sw, ip, port="6633", ptcp="" ):
         """
-           count is only needed if there is more than 1 controller"""
-        args = utilities.parse_args( [ "COUNT" ], **kwargs )
-        count = args[ "COUNT" ] if args != {} else 1
-
-        argstring = "SW"
-        for j in range( count ):
-            argstring = argstring + ",IP" + \
-                str( j + 1 ) + ",PORT" + str( j + 1 )
-        args = utilities.parse_args( argstring.split( "," ), **kwargs )
+        Description:
+            Assign switches to the controllers ( for ovs use only )
```

Comment:
I agree, almost every function here is for ovs only. We should include other type of switches in our plan in the following week. Curious what other types are there that we can virtually test?